app/src/main/java/com/vci/vectorcamapp/core/data/dto/cache/CurrentSessionCacheDto.kt

```diff
@@ -24,7 +24,8 @@ data class CurrentSessionCacheDto(
     val latitude: Float? = null,
     val longitude: Float? = null,
     @Serializable(with = SessionTypeSerializer::class)
-    val type: SessionType = SessionType.SURVEILLANCE
+    val type: SessionType = SessionType.SURVEILLANCE,
+    val collectorLastTrainedOn: Long? = null,
```

Comment:
Same here. Also, I think this can be a Long with default value 0L instead of Long?